apps/ideaspace/src/components/modules/DashboardPage/Ideas/IdeaCard.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import Link from 'next/link';
+import { atoms } from '@devlaunchers/components/src/components';
+import {
+  ImgButton,
+  StatuBox,
+} from './StyledIdeaCard';
+
+function IdeaCard({ cards }) {
+  return (
+    <atoms.Box flexDirection='column'>
+
+      <Link href={`/ideaspace/workshop/${cards.id}`}>
+        <ImgButton>
+          <img
+            alt="idea_image"
+            src={`https://picsum.photos/seed/${cards.id}/360/270?random=${cards.id}`}
+            style={{ width: "100%", height: "100%" }}
+          />
+
+          {/*cards.statu == "published" ? (
+            <StatuBox style={{background: "#C0E1EB",}}>published</StatuBox>
+          ) : (
+            <StatuBox style={{background: "#FFD580",}}>pending</StatuBox>
+          )*/}
+        </ImgButton>
+      </Link>
+
+
+      <atoms.Box justifyContent='space-between' alignItems='center' margin='0.5rem 0 0'>
+        <atoms.Typography type='h5' style={{ fontSize: '1.3rem', textAlign: 'left', width: '15rem' }}>
+          {cards.ideaName}
+        </atoms.Typography>
+{/* 
+        <Link href={`/ideaspace/edit/${cards.id}`}>
+          <atoms.Button
+            buttonSize='standard'
+            buttonType='primary'
+            style={{ width: '7.2rem' }}
+          >
+            EDIT&nbsp;DATA
+          </atoms.Button>
+        </Link>*/}
+      </atoms.Box>
+
+      {cards.comments.length > 0 ? (
+        <atoms.Typography type='h4' style={{ fontSize: '1.2rem', textAlign: 'left', marginTop: '-1rem' }}>
+          Comments:&nbsp;{cards.comments.length}
+        </atoms.Typography>
+      ) : (
+        <br/>
```

Comment:
Can we avoid using this `<br/>` tag and instead use styling?

Source: https://stackoverflow.com/questions/14197709/is-it-out-of-date-to-use-br-tag-nowadays-if-so-what-should-i-use-instead

https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br

Reply:
> Can we avoid using this `<br/>` tag and instead use styling?
> 
> Source: https://stackoverflow.com/questions/14197709/is-it-out-of-date-to-use-br-tag-nowadays-if-so-what-should-i-use-instead
> 
> https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br

We can't use styling. Maybe we can use  
`<atoms.Typography type='h4' style={{ fontSize: '1.2rem', marginTop: '-1rem' }}>&nbsp;</atoms.Typography>` . 
But I thought  `<br /> ` is shorter and more readable.

Reply:
Okay, I'm not sure I understand the purpose of this element. I don't think it is necessary and we can use `null` instead. I replaced it and the display of this component did not seem to change:

with `<br />`
<img width="1105" alt="Screen Shot 2022-12-29 at 10 10 16 AM" src="https://user-images.githubusercontent.com/43216378/209979744-e79010dd-2f6b-4f1f-9a42-03b104a0a723.png">

with `null`
<img width="919" alt="Screen Shot 2022-12-29 at 10 10 41 AM" src="https://user-images.githubusercontent.com/43216378/209979757-702d29fd-a2f7-4b82-9d44-15460e4e8f37.png">

code change:
<img width="889" alt="Screen Shot 2022-12-29 at 10 12 15 AM" src="https://user-images.githubusercontent.com/43216378/209979861-15902370-bd7f-4038-beb1-fe9b349d6004.png">


Here's some background for this approach:
https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering

https://stackoverflow.com/questions/47192353/best-practice-for-an-empty-component-null-empty-div-a-valid-react-element

Reply:
> Okay, I'm not sure I understand the purpose of this element. I don't think it is necessary and we can use `null` instead. I replaced it and the display of this component did not seem to change:
> 
> with `<br />` <img alt="Screen Shot 2022-12-29 at 10 10 16 AM" width="1105" src="https://user-images.githubusercontent.com/43216378/209979744-e79010dd-2f6b-4f1f-9a42-03b104a0a723.png">
> 
> with `null` <img alt="Screen Shot 2022-12-29 at 10 10 41 AM" width="919" src="https://user-images.githubusercontent.com/43216378/209979757-702d29fd-a2f7-4b82-9d44-15460e4e8f37.png">
> 
> code change: <img alt="Screen Shot 2022-12-29 at 10 12 15 AM" width="889" src="https://user-images.githubusercontent.com/43216378/209979861-15902370-bd7f-4038-beb1-fe9b349d6004.png">
> 
> Here's some background for this approach: https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering
> 
> https://stackoverflow.com/questions/47192353/best-practice-for-an-empty-component-null-empty-div-a-valid-react-element

no. it can work because the ideacards near it has comment. 
`<br />` works on thin screen and whole line without comment.
![1](https://user-images.githubusercontent.com/33647003/210011067-6428ac17-798e-4ff7-81a4-51a97309ad8b.JPG)
![3](https://user-images.githubusercontent.com/33647003/210011077-7396664a-9268-4a2a-a80a-b53671ea177a.JPG)